posts/How-to-Install-Memgraph-and-Memgraph-Lab-on-Windows-10

```diff
@@ -0,0 +1,179 @@
+ 
+Once you have made sure you are running the 64-bit version of Windows, you need to enable the Windows subsystem for Linux. To fo this, you can follow [these instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10). Please make sure to install the **Debian** distribution of Linux from the Windows App store.
+ 
+Once your installation is complete, you can test it by clicking on the **start** button on the bottom left side of your Windows screen, searching for **Debian** and launching the app.
+ 
+![Debian Launcher in Windows Start](https://i.imgur.com/NS8e0T9.png)
+ 
+The first time you launch the app, you will be asked to enter your username and password. If everything works properly, you will get the following output:
+ 
+![Debian shell running in Windows](https://i.imgur.com/6XYwxg5.png)
+ 
+Congratulations! You have successfully installed the Debian distribution of Linux on your Windows machine. You are now ready to install Memgraph.
+## Step 2 — Installing Memgraph
+ 
+To get started, go to Memgraph's [downolad](https://memgraph.com/download/) page and download the latest Debian installation package. 
+ 
+Once the download is complete, open your Debian shell and run the following command to start the installation process:
+```
+dpkg -i /path/to/memgraph_<version>.deb
+```
+* replace `/path/to` with path where you downloaded your installation package.
+* replace `_version` with the version of the package that you are installing(usually the name of the installation package you downloaded). 
+ 
+For example, if you downloaded version `0.1.14.1-1_amd64` to your D: Directory than this command will be changed to:
+```
+dpkg -i /mnt/d/memgraph_0.14.1-1_amd64.deb
+```
+Note: If you see any error related to missing dependency packages. You might have to run `sudo apt get update` before installing Memgraph.
+ 
+<p id="run-memgraph-server">Now, start the Memgraph server by issuing the following command:</p>
+ 
+```
+/usr/lib/memgraph/memgraph
+```
+If Memgraph has been installed correctly you will see something like this:
+> Starting 4 BoltS workers
+> BoltS server is fully armed and operational
+> BoltS listening on 0.0.0.0:7687
+ 
+Awesome! You now have a running instance of Memgraph on your Windows machine. In the next step, you will learn how to query Memgraph from the command line interface (CLI).
+ 
+## Step 3 — Querying Memgraph through the CLI tool with mg_client.
+ 
+Memgraph comes with a tool called `mg_client` which allows you to execute queries against the Memgraph server from your command-line interface.
+ 
+To install `mg_client`, you will need to open a new Debian shell and run the following command:
+```
+mg_client
+```
+If the mg_client has successfully connected to the server, you’ll receive a prompt similar to this:
```

Comment:
This is obsolete. To query Memgraph from the terminal, users should use https://github.com/memgraph/mgconsole.